examples-gallery/plot_parallel_park.py

```diff
@@ -223,8 +228,18 @@
 coords = []
 for xi, ui in zip(xs.T, us.T):
     coords.append(eval_point_coords(xi, ui, list(par_map.values())))
-coords = np.array(coords)  # shape(600, 3, 8)
-
+coords = np.array(coords)  # shape(501, 3, 5)
+
+coords1 = []
+time1 = []
+for i in range(coords.shape[0]):
+    if i % 3 == 0:
+        coords1.append(coords[i, :, :])
+        time1.append(time[i])
+coords1.append(coords[-1, :, :])
+time1.append(time[-1])
+time = np.array(time1)
+coords = np.array(coords1)
```

Comment:
This is an overly complicated way to skip frames (if that is what it does). It isn't so favorable for such things to be in examples for beginners.

Reply:
Just use slicing in `numpy` that would be a need approach. Though an easier approach in this case would be providing a range like `range(0, len(time), 3)` to `FuncAnimation`.

Reply:
> This is an overly complicated way to skip frames (if that is what it does). It isn't so favorable for such things to be in examples for beginners.

Yes, I wanted to skip frames, and I could not think of a simpler way.

Reply:
I thought one of the examples already has exactly what Timo suggests.

Reply:
> Just use slicing in `numpy` that would be a need approach. Though an easier approach in this case would be providing a range like `range(0, len(time), 3)` to `FuncAnimation`.

Not sure, I understand what slicing means - my Python skill are limited. Could you explain, please.
This "range thing" I understand, I will try!

Reply:
> I thought one of the examples already has exactly what Timo suggests.
I will look at the examples for this feature, I did not look before.

Reply:
> Internet search of "numpy slicing" points to: https://numpy.org/doc/stable/user/basics.indexing.html#slicing-and-striding

Now clear! This I even know, just did not think about it :-(

Reply:
@Peter230655 in this case it is even easier to use `ani = animation.FuncAnimation(fig, animate, range(0, len(time), 3),` than numpy slicing. Though learning more about numpy slicing is definitely useful.

Reply:
> @Peter230655 in this case it is even easier to use `ani = animation.FuncAnimation(fig, animate, range(0, len(time), 3),` than numpy slicing. Though learning more about numpy slicing is definitely useful.

I will try this first thing tomorrow morning!! :-)